rbac/management/workspace/view.py

```diff
@@ -52,6 +45,11 @@ class WorkspaceViewSet(BaseV2ViewSet):
     ordering = ("name",)
     filter_backends = (filters.DjangoFilterBackend, OrderingFilter)
 
+    def __init__(self, **kwargs):
+        """Init viewset."""
+        super().__init__(**kwargs)
+        self._service = WorkspaceService()
```

Comment:
When we initialize it we haven't put any replicator in it. Would it cause issue

Reply:
Updated per conversation to remove from the constructor for now.